### ISSUE TITLE
Colour header according to visibility of ML

### ISSUE THREAD

**Issue by tripu** [w3c/mailing-list-archives#19]
When #12 is merged:
Use the new fixed header to carry information about the _visibility_ of the message (ie, of the mailing list). Apply the same colours we have in other parts of the site (see images).

![public](https://cloud.githubusercontent.com/assets/1016538/13564249/927263ae-e48b-11e5-9391-fdfeb9af03a0.png)
![member](https://cloud.githubusercontent.com/assets/1016538/13564251/97b0b4d8-e48b-11e5-8ad2-da70ac03f593.png)
![team](https://cloud.githubusercontent.com/assets/1016538/13564252/9bad38fe-e48b-11e5-8666-13e0081d899d.png)

I think that will be more consistent with the design principles of 2008. And as the header is fixed, that information will be visible permanently (the header doesn't go away).

Changing the background colour of the entire page to one of those three pastel hues, like we do now, is ugly; I think we grew so used to it that we don't _see_ it anymore, but it doesn't look good, imho.

**Comment by gbaudusseau** [comment]
yes pastel hues as a background for the entire page is ugly. Let's keep it simple and only for the header

**Comment by tguild** [comment]
We have made some horrible color choices in the past.  I am fine with shedding those for a background and using some other visual indicator like icons (that can use non-puke colors)
